cpp/include/Ice/Object.h

```diff
@@ -136,7 +118,7 @@ class ICE_API Blobject : public virtual Object
     virtual bool ice_invoke(std::vector<std::uint8_t> inEncaps, std::vector<std::uint8_t>& outEncaps, const Current& current) = 0;
 
     /// \cond INTERNAL
-    virtual bool _iceDispatch(IceInternal::Incoming&, const Current&);
+    virtual bool _iceDispatch(IceInternal::Incoming&);
```

Comment:
The separate current parameter was a misfeature as its memory is owned by the incoming parameter. This relationship became unclear when I moved incoming into a new heap-allocated shared incoming.